packages/table/src/table.vue

```diff
@@ -39,6 +40,9 @@
       <div :style="{ width: bodyWidth }" class="el-table__empty-block" v-if="!data || data.length === 0">
         <span class="el-table__empty-text"><slot name="empty">{{ emptyText || t('el.table.emptyText') }}</slot></span>
       </div>
+      <div class="el-table__append-wrapper" ref="appendWrapper" v-if="$slots.append">
+        <slot name="append"></slot>
+      </div>
```

Comment:
@syn-zeta This slot is moved outside the table body, so the content shouldn't be nested in `<tr>` and `<td>`:https://jsfiddle.net/4ez4j2j4/1/ . Looks we should address this in the breaking change @liril-net